src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -8939,10 +8939,12 @@ static bool isCandidateUnique(ref MethodSymbol? method, MethodSymbol candidate)
             TypeWithAnnotations returnType,
             ImmutableArray<RefKind> parameterRefKinds,
             ImmutableArray<DeclarationScope> parameterScopes,
-            ImmutableArray<TypeWithAnnotations> parameterTypes)
+            ImmutableArray<TypeWithAnnotations> parameterTypes,
+            ImmutableArray<ConstantValue?> parameterDefaultValues = default)
```

Comment:
> = default [](http://example.com/codeflow?start=65&length=10)

There are only two callers of `GetMethodGroupOrLambdaDelegateType()`, and both will likely support default values, so we let's remove the default value for this parameter.